docs/book/driver-deployment/upgrade.md

```diff
@@ -0,0 +1,30 @@
+<!-- markdownlint-disable MD033 -->
+# vSphere CSI Driver - Upgrade
+
+**Note: To upgrade "vSphere with Tanzu" follow vSphere documentation. Following instructions are applicable to native vanilla Kubernetes Cluster deployed on vSphere.**
+
+Each release of vSphere CSI driver has a different set of RBAC rules and deployment YAML files depending on the version of the vCenter on which Kubernetes Cluster is deployed.
+
+To upgrade from one release to another release of vSphere CSI driver, we recommend to completely delete the driver using the original YAML files, you have used to deploy the driver and then use new YAML files to re-install the vSphere CSI driver.
+
+For example, if you want to upgrade driver from `v2.0.0` release to `v2.0.1` release deployed on the Kubernetes Cluster installed on `vSphere 67u3` release
+
+Uninstall `v2.0.0` driver
+
+```bash
+kubectl delete -f https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/v2.0.0/vsphere-67u3/vanilla/rbac/vsphere-csi-controller-rbac.yaml
```

Comment:
Shouldn't the URLs be the raw content? Ex: https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/manifests/v2.0.0/vsphere-67u3/vanilla/rbac/vsphere-csi-controller-rbac.yaml

Reply:
This URL also works for me.

Reply:
but let me change it to raw URL to be safe.

Reply:
used raw URL - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/439/commits/c5d41501c331d2a6739c927a3cac9f7868dac606

if change looks good, I will squash commits.